package.json

```diff
@@ -5,10 +5,15 @@
   "repository": "stephenplusplus/gcp-metadata",
   "main": "index.js",
   "files": [
-    "index.js"
+    "index.js",
+    "index.d.ts"
   ],
+  "types": "index.d.ts",
   "scripts": {
-    "test": "standard && mocha *.test.js"
+    "compile": "tsc",
+    "clean": "rm index.js index.d.ts",
+    "prepare": "npm run compile",
+    "test": "npm run compile && standard && mocha *.test.js"
```

Comment:
`standard` (the style rules) was removed from this... is it possible to keep using it with TypeScript?

Reply:
The question would be whether standard works (well) for typescript code?

For all code in Google Cloud repos, we are starting to use the the [Google TS style](https://npm.im/gts), but for this repo you can decide what you prefer.

Reply:
It's fine to go with the Google TS style. Just remove the `standard` dep.